examples/azure-rules.yaml

```diff
@@ -656,4 +656,17 @@ groups:
           Please, check the naming convention in case the resource configuration is done correctly.
            
           More information at https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources
-        scored: true
\ No newline at end of file
+        scored: true
+      - id: 1.5.7
+        description: "Check /hana/share/myHooks/global.ini"
+        audit: 'cat $(find /hana/shared -name global.ini | grep "global/hdb/custom/config/global.ini" | head -n1)'
+        tests:
+          test_items:
+          - flag: "provider = SAPHanaSR"
+          - flag: "path = /usr/share/SAPHanaSR"
+          - flag: "execution_order = 1"
+          - flag: "ha_dr_saphanasr = info"
```

Comment:
I'm not entirely clear how does this check work.
Why are we first finding multiple config files and then scanning only the first one?

Also, how do multiple `test_items` are combined? Does it mean that it looks for all of them?

Reply:
> I'm not entirely clear how does this check work.
> Why are we first finding multiple config files and then scanning only the first one?

The file is stored in /hana/shared/<SID>/global/hdb/custom/config/ but we cannot parametrise the SID, so it takes for the first SID found

> Also, how do multiple `test_items` are combined? Does it mean that it looks for all of them?

It checks that the file has all those strings "provider = SAPHanaSR" AND "path = /usr/share/SAPHanaSR" AND ...

Reply:
A simpler approach that could work is to repeat the check of ALL the SIDs, but you need to ask Lee if that would work. I'm not sure about how the shared storage might be used in the context of multiple systems.

Reply:
@lee-martin we need your advice.